site/docs/Contributing/implemenation.mdx

```diff
@@ -19,6 +19,30 @@ import Link from "../../src/components/Link";
 
 ### Guidelines for implementation
 
-_HDS specific guidelines are coming soon!_
+<li>HDS follows dev.hel.fi fontend coding standards: <Link href="https://dev.hel.fi/coding-standards#frontend-code-standards" external>Frontend coding standards</Link></li>
 
-<p>Take a look at <Link href="https://dev.hel.fi/" external>Develop with Helsinki site</Link> for practical details for developing open source code for the City of Helsinki.</p>
\ No newline at end of file
+#### HDS Technologies
+
+<li>HDS React and HDS Core both apply the <Link href="https://storybook.js.org/">storybook library</Link></li>
+<li>HDS React is build with Typescript and SASS</li>
+<li>HDS Core is build with pure JS and CSS</li>
+<li>HDS applies yarn as its dependency manager</li>
+<li>HDS applies git for version control with <Link href="https://git-lfs.github.com/">GIT-LFS support</Link></li>
+
+#### Creating a new component
+
+<li>Apply the Scaffolding yarn script when creating a new component: <Link href="https://github.com/City-of-Helsinki/helsinki-design-system/blob/master/packages/react/package.json#L17">Scaffolding</Link></li>
+<li>For simple components, create both versions, HDS Core and HDS React. Create common styles to HDS core that you can then import to HDS React. HDS React has HDS Core as its dependency</li>
+<li>Add ready components to <Link href="https://github.com/City-of-Helsinki/helsinki-design-system/blob/master/packages/react/rollup.config.js">rollup.config.js</Link> (We should probably add this to scaffold script?)</li>
+
+#### Testing
+
+<li>Write unit tests for each component. Test rendering, accessibility and possible functionality of the component. HDS applies <Link href="https://jestjs.io/">Jest</Link> for unit testing </li>
+<li>HDS applies <Link href="https://loki.js.org/">Loki</Link> for visual regression testing. Loki is integrated with storybook. Hence it's important to add a story for each different UI state of the component</li>
+<li>It is a good idea to test the component independently in a fresh <Link href="https://github.com/City-of-Helsinki/hds-cra">Create React App</Link></li>
+
+#### General guides
```

Comment:
Jos joku lukee näitä järjestyksessä, niin nämä olisi ehkä hyvä olla jo ensimmäisenä listassa. Näistä osa on kuitenkin sellaisia jotka olisi hyvä tietää alusta alkaen.

Reply:
Hyvä idea:

- [ ] Siirrä _General guides_ ensimmäiseksi